User.py

```diff
@@ -95,3 +95,14 @@ def addExercise(self, exercise, reps):
     def hasDoneExercise(self, exercise):
         return exercise["id"] in self.exercise_counts
 
+
+def parseSlackJSON(response):
+    isMessageOkay = False
+    try:
+        if response.ok:
+            parsed_message = json.loads(response.text, encoding='utf-8')
+    except:
+        print "Caught exception parsing response status: " + str(response.status_code) + ", text: " + response.text
```

Comment:
Here you should return null, False so you don't get to the other part in the case that `response.ok` is True but we were unable to parse the json